pkg/client/openstack/kluster/client.go

```diff
@@ -73,24 +78,39 @@ func (c *klusterClient) CreateNode(kluster *v1.Kluster, pool *models.NodePool, n
 			{UUID: kluster.Spec.Openstack.NetworkID},
 		}
 	}
+	flavorID, err := flavorutil.IDFromName(c.ComputeClient, pool.Flavor)
+	if err != nil {
+		return "", fmt.Errorf("Failed to find id for flavor %s: %w", pool.Flavor, err)
+	}
+	imageID, err := imageutil.IDFromName(c.ImageClient, pool.Image)
+	if err != nil {
+		return "", fmt.Errorf("Failed to find id for image %s: %w", pool.Image, err)
+	}
+
+	tags := nodeTags(kluster.Spec.Name, pool.Name)
+	tags = append(tags, "kubernikus:template-version="+templates.TEMPLATE_VERSION)
+	tags = append(tags, "kubernikus:api-version="+kluster.Spec.Version)
+	metadata := nodeMetadata(kluster.Spec.Name, pool.Name)
```

Comment:
Is `kluster.Name` == `kluster.Spec.Name`?

Reply:
No, that's the unqualified name (without the project id), so the tag will be `kubernikus:kluster=s-qa-de-1` not `kubernikus:kluster=s-qa-de-1-10397130u4u10140109470921471024`. I figured its just noise as the server is running in the project anyways.

Reply:
Cool. Open the flood-gates!